truenas/webui
# NAS-119668: Correctly report vdev width

To test:

Create each configuration and check the reported width in the Storage Dashboard topology card.

- Two raidz1 vdevs with 5 disks in each vdev. Topology card should say "5 wide"
- Two mirror vdevs with 2 disks in each vdev. Topology card should say "2 wide"
- Two mirrors. One made with two disks and another with 3 disks. Topology card should say "Mixed Width VDEVs"
